### PR TITLE
From glib unsafe

### LINKED ISSUE #197: from_glib (both method and trait) should be unsafe

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #197.
Regen from https://github.com/gtk-rs/gir/pull/1018

### CODE REVIEW

**Comment by sdroege** [comment]
You missed quite some code, see the CI :)

File: graphene/sys/versions.txt [hunk 1]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
 Generated by gir (https://github.com/gtk-rs/gir @ 3918b7c)
+=======
```
Comment (sdroege) [inline graphene/sys/versions.txt:3]: You probably didn't want this :)

File: atk/src/auto/versions.txt [hunk 2]
```diff
@@ -1,2 +1,6 @@
+<<<<<<< HEAD
 Generated by gir (https://github.com/gtk-rs/gir @ 3918b7c)
+=======
+Generated by gir (https://github.com/gtk-rs/gir @ 20cdbb3+)
+>>>>>>> regen
```
Comment (sdroege) [inline atk/src/auto/versions.txt:5]: And here and in other places

File: graphene/sys/versions.txt (hunk 1 above)
Comment (GuillaumeGomez) [reply graphene/sys/versions.txt:3]: I intend to remove this commit and regen once again. :)

**Comment by GuillaumeGomez** [comment]
Ok, seems like we're all good here now! \o/

**Review by sdroege — APPROVED** [review]
Looks good, please update gir another time and then regenerate again :)

Can you also send an MR for gstreamer and/or gtk4?

**Comment by GuillaumeGomez** [comment]
> Can you also send an MR for gstreamer and/or gtk4?

Sure!

**Pull request merged**